stdlib/asyncio/events.pyi

```diff
@@ -451,21 +451,37 @@ class AbstractEventLoop:
             ssl: _SSLContext = ...,
         ) -> Server: ...
 
-    @abstractmethod
-    async def create_datagram_endpoint(
-        self,
-        protocol_factory: Callable[[], _ProtocolT],
-        local_addr: tuple[str, int] | None = ...,
-        remote_addr: tuple[str, int] | None = ...,
-        *,
-        family: int = ...,
-        proto: int = ...,
-        flags: int = ...,
-        reuse_address: bool | None = ...,
-        reuse_port: bool | None = ...,
-        allow_broadcast: bool | None = ...,
-        sock: socket | None = ...,
-    ) -> tuple[BaseTransport, _ProtocolT]: ...
+    if sys.version_info >= (3, 11):
+        @abstractmethod
+        async def create_datagram_endpoint(
+            self,
+            protocol_factory: Callable[[], _ProtocolT],
+            local_addr: tuple[str, int] | None = ...,
+            remote_addr: tuple[str, int] | None = ...,
+            *,
+            family: int = ...,
+            proto: int = ...,
+            flags: int = ...,
+            reuse_port: bool | None = ...,
```

Comment:
On freshly pulled cpython's `main` branch, I get:

```
akuli@akuli-desktop:~/sourcestuff/cpython$ grep -A4 'def create_datagram_endpoint' Lib/asyncio/events.py 
    async def create_datagram_endpoint(self, protocol_factory,
                                       local_addr=None, remote_addr=None, *,
                                       family=0, proto=0, flags=0,
                                       reuse_address=None, reuse_port=None,
                                       allow_broadcast=None, sock=None):
```

So it seems like `reuse_address` was not removed. Am I missing something?